### PR TITLE
Stream: docs for stream-media-id support

### PR DESCRIPTION
**Pull request by zaidf**

### CODE REVIEW

File: products/stream/src/content/uploading-videos/upload-video-file.md [hunk 1]
```diff
@@ -81,6 +81,24 @@ Setting arbitrary metadata values in the `Upload-Metadata` header sets values th
 
 </Definitions>
 
+### Getting the video id when using tus
+
+When you make the initial tus request, it responds with a tokenized URL in the location header. However, while the tokenized URL may contain the video id, this is not guaranteed behavior. We do not recommend parsing the tokenized URL to get the video id.
```
Comment (Everlag) [inline products/stream/src/content/uploading-videos/upload-video-file.md]: We're not consistent about it elsewhere in the docs but you may want to capitalize `ID` in `video id`; that also applies to the next line.

**Review by Everlag — APPROVED** [review]
One non-blocking comment on capitalization, otherwise LGTM.

File: products/stream/src/content/uploading-videos/upload-video-file.md (hunk 1 above)
Comment (renandincer) [reply products/stream/src/content/uploading-videos/upload-video-file.md]: same with video id vs media id

File: products/stream/src/content/uploading-videos/upload-video-file.md [hunk 2]
```diff
@@ -81,6 +81,24 @@ Setting arbitrary metadata values in the `Upload-Metadata` header sets values th
...
+
+Instead, Stream returns a custom header named `Stream-Media-ID` which contains the video id. You should develop your application so it reads the `Stream-Media-ID` property in the response headers.
+
+**Here's a sample response header which shows the stream-media-id property:**
```
Comment (renandincer) [inline products/stream/src/content/uploading-videos/upload-video-file.md]:
```suggestion
<Example>
```

File: products/stream/src/content/uploading-videos/upload-video-file.md [hunk 3]
```diff
@@ -81,6 +81,24 @@ Setting arbitrary metadata values in the `Upload-Metadata` header sets values th
...
+```
+location: https://upload.videodelivery.net/tus/cab807e0c477d01baq20f66c3d1dfc26cf
+...
+stream-media-id: cab807e0c477d01baq20f66c3d1dfc26cf
+tus-resumable: 1.0.0
+x-content-type-options: nosniff
+x-envoy-upstream-service-time: 260
+server: cloudflare
+```
+
```
Comment (renandincer) [inline products/stream/src/content/uploading-videos/upload-video-file.md]:
```suggestion
</Example>
```

File: products/stream/src/content/uploading-videos/upload-video-file.md [hunk 4]
```diff
@@ -81,6 +81,24 @@ Setting arbitrary metadata values in the `Upload-Metadata` header sets values th
 
 </Definitions>
 
+### Getting the video id when using tus
```
Comment (renandincer) [inline products/stream/src/content/uploading-videos/upload-video-file.md]:
```suggestion
### Getting the video ID when using TUS
```

File: products/stream/src/content/uploading-videos/upload-video-file.md (hunk 1 above)
Comment (renandincer) [inline products/stream/src/content/uploading-videos/upload-video-file.md]:
```suggestion
When an initial TUS request is made, Stream responds with a URL in the location header. While this URL may contain the video ID, it is not recommend to parse this URL to get the ID.
```

File: products/stream/src/content/uploading-videos/upload-video-file.md [hunk 5]
```diff
@@ -81,6 +81,24 @@ Setting arbitrary metadata values in the `Upload-Metadata` header sets values th
...
+
+When you make the initial tus request, it responds with a tokenized URL in the location header. However, while the tokenized URL may contain the video id, this is not guaranteed behavior. We do not recommend parsing the tokenized URL to get the video id.
+
+Instead, Stream returns a custom header named `Stream-Media-ID` which contains the video id. You should develop your application so it reads the `Stream-Media-ID` property in the response headers.
```
Comment (renandincer) [inline products/stream/src/content/uploading-videos/upload-video-file.md]:
```suggestion
Instead, the `Stream-Media-ID` HTTP header in the response should be used to retrieve the video ID.
```

**Review by renandincer — CHANGES_REQUESTED** [review]

**Review by renandincer — APPROVED** [review]

**Pull request merged**